packages/nc-gui/components/general/Overlay.vue

```diff
@@ -49,8 +50,10 @@ export default {
         vModel ? 'opacity-100' : 'opacity-0 pointer-events-none',
         inline ? 'absolute' : 'fixed',
         transition ? 'transition-opacity duration-200 ease-in-out' : '',
+        lightBackground ? 'bg-gray-100/25' : '',
```

Comment:
2. so that we can just use `lightBackground?  '!bg-gray-100/25': '',` to override the dark overlay.